test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -332,16 +392,6 @@ describe.skip('Create token, approve token and approve token without gas', funct
             text: 'Approve Token spend limit',
           });
 
-          await driver.clickElement('.transaction-list-item');
-
-          const permissionInfo = await driver.findElements(
-            '.confirm-approve-content__medium-text',
-          );
-          const recipientDiv = permissionInfo[1];
-          assert.equal(await recipientDiv.getText(), '0x2f318C33...C970');
-
-          await driver.clickElement({ text: 'Confirm', tag: 'button' });
-
           await driver.waitForSelector({
             css: '.transaction-list__completed-transactions .transaction-list-item:first-child .list-item__heading',
             text: 'Approve Token spend limit',
```

Comment:
I think finishing each test case with an assertion is a good convention. It could be added in this last testcase

Reply:
Done